### PR TITLE
Add log ui page

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Jira
https://jira.mongodb.org/browse/DOCSP-21239

### Staged Changes (Requires MongoDB Corp SSO)

https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/docsp-21239/logs/logs-ui/

### CODE REVIEW

File: source/logs/logs-ui.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+On the Logs screen, you can do the following:
+
+**Filter the log entries**
+   Drop-down menus enable filtering by predefined types of log entry, the status 
+   of entries. You can also specify a date range, filter by user ID, and show 
+   only entries associated with a specific request ID. 
+
+**Limit the number of log entries**
+   In the text field with the placeholder text :guilabel:`Max # of Logs`, enter 
+   the maximum number of results you want to display.
```
Comment (dacharyc) [inline source/logs/logs-ui.txt:34]: "the maximum number of results you want to display" makes me wonder - is this paginated? Is it # of results per page or total number of results to display?

**Review by dacharyc — APPROVED** [review]
LGTM! Just a small Q that may or may not warrant clarifying in the docs.

File: source/logs/logs-ui.txt [hunk 2]
```diff
@@ -0,0 +1,44 @@
+=======================
```
Comment (dacharyc) [inline source/logs/logs-ui.txt:1]: Oops, looks like these are causing a Snooty build error. Does Snooty count it based on the expanded source constant?

File: source/logs/logs-ui.txt (hunk 1 above)
Comment (MongoCaleb) [reply source/logs/logs-ui.txt:34]: Great question -- turns out, it's pagination.

File: source/logs/logs-ui.txt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+You can view, filter, search, and download an application's logs in the {+ui+}. 
+In the left-hand navigation, select :guilabel:`Logs`:
```
Comment (dacharyc) [inline source/logs/logs-ui.txt:17]: One more tiny nit - I noted this before but didn't comment, but then I see that _you_ commented on this in another PR this week, so I'm mentioning it ;) 

Might we want to point out that the option the reader is looking for is under :guilabel:`Manage`? i.e. 
> In the left-hand navigation, under :guilabel:`Manage`, select :guilabel:`Logs`:

**Pull request merged**